avtalia.lic

```diff
@@ -0,0 +1,34 @@
+custom_require.call(%w[common common-arcana])
```

Comment:
💡 Please add the documentation header block

Reply:
I'll have to put something out there to refer to later, but it's done.